service/src/main/java/uk/gov/hmcts/reform/fpl/service/respondent/RespondentValidator.java

```diff
@@ -48,21 +52,22 @@ public List<String> validate(CaseData caseData, CaseData caseDataBefore) {
         errors.addAll(validateEmailService.validate(emails, "Representative"));
 
         if (caseData.getState() != OPEN) {
-            errors.addAll(respondentAfterSubmissionValidator.validate(caseData, caseDataBefore));
+            errors.addAll(respondentAfterSubmissionValidator.validate(caseData, caseDataBefore, hideRespondentIndex));
         }
 
         return errors;
     }
 
-    private List<String> validateDob(CaseData caseData) {
+    private List<String> validateDob(CaseData caseData, boolean convertedFromOther) {
```

Comment:
think the arg should be hideRespondentIndex

Reply:
Done